internal/guestrequest/types.go

```diff
@@ -30,7 +30,12 @@ type LCOWMappedVirtualDisk struct {
 	ReadOnly   bool
 }
 
-// Plan 9.
+type WCOWMappedVirtualDisk struct {
```

Comment:
What cant you just use schema2.MappedVirtualDisk?

Reply:
As it's no longer generated. It was previously pulled in through attachments, but that changed, hence this structure has gone.